dev/fb/helmfile.yaml

```diff
@@ -28,23 +28,21 @@ releases:
     namespace: fb   # target namespace
     createNamespace: true
     chart: datarepo-helm/datarepo-api   # the chart name
-    version: 0.0.25    # chart version
     missingFileHandler: Warn
     values:
       - datarepo-api.yaml   # Value files passed via --values
   - name: fb-jade-datarepo-ui   # name of this release
     namespace: fb   # target namespace
     createNamespace: true
     chart: datarepo-helm/datarepo-ui   # the chart name
-    version: 0.0.14    # chart version
     missingFileHandler: Warn
     values:
       - datarepo-ui.yaml   # Value files passed via --values
```

Comment:
Removing this version will default to the latest api & ui charts. We could do it for all of the charts, but Mark recommended just doing this for api & ui.